### PR TITLE
Update Readme.md to specify operating system compatibility in Virtual x86

### PR DESCRIPTION
**Pull request by spetterman66**
Read the title to see what this PR is about :)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -67,7 +67,13 @@ list of emulated hardware:
...
 
 - Linux works pretty well. 64-bit kernels are not supported.
-  - Damn Small Linux (2.4 Kernel) works.
+  - Damn Small Linux (2.4.31 kernel) works.
+  - KNOPPIX 9.1 works.
+  - Elementary OS 0.3.2 works.
+  - Slax works.
+  - AntiX works
+  - Fedora 30 works.
+  - Zorin OS 12.6 works.
```
Comment (copy) [inline Readme.md:76]: We don't really need to name every Linux distribution.  How about just Fedora, the is based on Debian or Ubuntu anyway.

**Comment by spetterman66** [comment]
yeah, good reason
________________________________

**Comment by spetterman66** [comment]
better?

**Comment by copy** [comment]
Looks good now. That said, does Fedora 30 *actually* work?

**Comment by spetterman66** [comment]
yes. see the screenshots i posted in the gitter discussion channel

**Comment by copy** [comment]
I can't reproduce it, does only the LXDE version work?

**Comment by spetterman66** [comment]
what specifications do you allocate to the virtual machine? put it on 2048 mb ram, 128 mb vram, and acpi off (not needed). boot arguments loglevel=7 earlyprintk=serial,keep for best verbosity and to get info where it gets stuck at

**Comment by spetterman66** [comment]
As you can see in the screenshot below, Fedora 30 (i386, LXDE) works fine in Virtual x86. I have not tried other spins.
image

**Comment by copy** [comment]
Does it work correctly? I wouldn't expect any distro (not even bloated ones) to take more than a few minutes to boot.

**Comment by BelleNottelling** [comment]
> Does it work correctly? I wouldn't expect any distro (not even bloated ones) to take more than a few minutes to boot.

It looks like Fedora 30 just outright works (although **very** slowly unless you are using one of the optional desktop environments.
- `Fedora-Workstation-Live-i386-30-1.2.iso` Booted to the live image about 8.5 min, but once it got to that point performance was awful and it's not even worth attempting, moving the mouse would be met with 30+ second long delays.
- `Fedora-LXDE-Live-i386-30-1.2.iso` Got me to a complete LiveCD desktop in about 7 min and felt very usable in performance.
- `Fedora-LXQt-Live-i386-30-1.2.iso` Took about 6min to get the LiveCD to the desktop and once again felt very usable.
- `Fedora-Xfce-Live-i386-30-1.2.iso` Took about 10 min for the LiveCD to get to the desktop and again performance felt fine.

These were tested with `2048MB` of RAM and `64MB` of VRAM allocated in the VM. ACPI was disabled and in all cases I used the default live image option that I was presented with.

I'm guessing the *very* long boot time in that other screenshot was either because of the boot arguments that were used or potentially a slow host was being used. I am testing things with an `i7 13700K` so it's somewhat a best-case scenario for CPU performance.

**Comment by copy** [comment]
> Fedora-Workstation-Live-i386-30-1.2.iso Booted to the live image about 8.5 min, but once it got to that point performance was awful and it's not even worth attempting, moving the mouse would be met with 30+ second long delays.

Hmm, indeed. It's similarly slow in qemu-kvm, so let's assume v86 isn't at fault here.

**Pull request merged**